test/integration/node-labels-webhook.test.js

```diff
@@ -124,11 +124,6 @@ function ignoreQueryParams (pathAndQuery) {
   return url.parse(pathAndQuery, true).pathname
 }
 
-function readFixture (fixtureName) {
```

Comment:
Extract the `readFixture()` helper method into its own submodule, as more and more tests needed it.